oppia_android_test.bzl

```diff
@@ -1,5 +1,9 @@
-load("@rules_jvm_external//:defs.bzl", "artifact")
+"""
+Starlark rules (Macros) to setup the WORKSPACE in the Oppia-Android codebase.
```

Comment:
```suggestion
Central macros pertaining to setting up tests across the codebase.
```

This is specific to tests, not the WORKSPACE.

Reply:
done

Reply:
resolving on behalf of Ben as it's a nit change.